sdk/tables/azure-data-tables/CHANGELOG.md

```diff
@@ -1,7 +1,17 @@
 # Release History
 
-## 12.2.0-beta.1 (Unreleased)
+## 12.2.0-beta.1 (2021-08-12)
```

Comment:
Are there any new APIs introduced in the beta release? If not, should this just be a patch GA release?

Reply:
No new APIs were introduced, however, since this is fixing some pretty impactful bugs when using Cosmos wouldn't it be good to also publish a new beta with these changes?

Reply:
I have no trouble with just doing a patch release, I'm just curious about what should be done.

Reply:
Will make this a patch release after talking to Srikanta about it.